tests/test.js

```diff
@@ -128,44 +128,74 @@ describe('Maestro Config', function() {
             }.bind({ctx: this, done: done}));
         });
     });
+
+    /**
+     * Logging tests
+     **/
+    describe('Logging', function() {
+
+    });
 });
 
+function maestro_api_set_ip_address(ctx, interface, ip_address)
+{
+    let view = [{
+        dhcpv4: false,
+        if_name: "eth" + interface,
```

Comment:
would it make sense for 'interface' param to contain the full iface name, rather than trying to build an iface name string here?

Reply:
Yep! Fixed it. Will push again shortly